src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <div class="col-sm-12 col-lg-12">
+      <div>
+          <cd-table-actions class="btn-group"
```

Comment:
I saw that you were putting `margin` on the css. We should avoid that as much as we can and use the `m-1` or the [bootstrap spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/) for better consistency.